frontend/src/components/dashboard/aliases/MaskCard.module.scss

```diff
@@ -16,129 +16,109 @@
   }
 }
 
-.bar {
+.summary {
   display: flex;
   align-items: center;
-  gap: $spacing-sm;
+  position: relative;
   padding: $spacing-sm;
+  gap: $spacing-sm;
+  flex-wrap: wrap;
+
+  @media screen and #{$mq-md} {
+    gap: $spacing-sm;
+    flex-wrap: nowrap;
+  }
+
+  .label-editor-wrapper {
+    overflow: visible;
+
+    @media screen and #{$mq-md} {
+      // Same as .summary's gap:
+      padding-inline-end: $spacing-sm;
+      border-right: 1px solid $color-grey-10;
+    }
+  }
 
-  .summary {
-    flex: 1 1 auto;
-    // Override flex children's default `min-width: auto` to allow the summary
-    // to shrink instead of pushing out the collapse button:
-    min-width: 0;
+  .copy-button-wrapper {
     display: flex;
     align-items: center;
-    flex-wrap: wrap;
     position: relative;
+    overflow: hidden;
+    width: 90%;
 
+    @media screen and #{$mq-md} {
+      width: 100%;
+    }
     @media screen and #{$mq-md} {
       gap: $spacing-sm;
-      flex-wrap: nowrap;
     }
-
-    .label-editor-wrapper {
-      @media screen and #{$mq-md} {
-        // Same as .summary's gap:
-        padding-inline-end: $spacing-sm;
-        border-right: 1px solid $color-grey-10;
-      }
+    @media screen and (max-width: $screen-xs) {
+      overflow: hidden;
     }
+  }
 
-    .copy-button-wrapper {
-      display: flex;
-      align-items: center;
+  .copy-button {
+    display: flex;
+    align-items: center;
+    gap: $spacing-xs;
+    background-color: transparent;
+    color: $color-blue-50;
+    border-style: none;
+    font-family: $font-stack-firefox;
+    font-weight: 500;
+    border-radius: $border-radius-sm;
+    cursor: pointer;
+    overflow: inherit;
 
-      @media screen and #{$mq-md} {
-        gap: $spacing-sm;
-      }
-      @media screen and (max-width: $screen-xs) {
-        overflow: hidden;
-      }
+    &:hover {
+      color: $color-green-70;
     }
 
-    .copy-button {
```

Comment:
Hey @rafeerahman, it looks like lots of code was deleted and added, but am I correct in thinking that maybe code mostly just got moved around? If so, could you make my life a bit easier and described what actually changes you made? Something like "I added a wrapper with class `.something` around `.something-else` and changed the way it was positioned by <doing something>"? :)